packages/appkit/src/client/appkit.ts

```diff
@@ -349,7 +349,9 @@ export class AppKit extends AppKitBaseClient {
           } else if (reason === 'unverified_domain') {
             AlertController.open(ErrorUtil.ALERT_ERRORS.UNVERIFIED_DOMAIN, 'error')
           }
-        }
+        },
+        getActiveCaipNetwork: (namespace?: ChainNamespace) =>
+          ChainController.getActiveCaipNetwork(namespace)
```

Comment:
Same here, since the types are same, no need to create new function, we can bind the ChainController's fn
```suggestion
        getActiveCaipNetwork: ChainController.getActiveCaipNetwork(namespace)
```